### DISCUSSION TITLE
Discontinued?

### DISCUSSION THREAD

**Discussion by paigeadelethompson** [axaq/traviso.js#36] [General]
What is the intended future of traviso, I think a few people are wondering about this..

**Comment by axaq** [comment]
Hi there. Yes I wasn't actively updating it for a long time unfortunately due to my daily job weight and life in general :). But now with the help of coding agents, I started progressing and updating much more quickly. If you are still interested, let me know what you want to see next from traviso.js. I have one release coming for pixi.js v7 support and then another one with pixi v8 support to modernise the underlying code and so that people that are atill using old pixi versions can pick the traviso version that works for them. After that I am going to focus on adding new features.
